emails/2025-07-18.md

```diff
@@ -0,0 +1,21 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+It's time for my annual summer trip to my mother country, Canada. This is your last email for a couple weeks so please enjoy responsibly.
+
+### What's the Best Coding Agent?
+
+Agents need branches. Dolt is the [only SQL database with branches](https://www.dolthub.com/blog/2024-09-18-database-branches/). Dolt is [the database for agents](https://www.dolthub.com/blog/2025-03-17-dolt-agentic-workflows/). We even changed [the homepage](https://www.dolthub.com/). 
+
+As such, we're throwing ourselves into coding agents. [Which coding agent is best?](https://www.dolthub.com/blog/2025-07-15-best-coding-agent/) I devised two sample tasks and [pitted Gemini, Codex, Claude Code, and Cursor against each other](https://www.dolthub.com/blog/2025-07-15-best-coding-agent/). Every agent performed well except Gemini. Read [the article](https://www.dolthub.com/blog/2025-07-15-best-coding-agent/) for the final rankings.
+
+### Faster Merges
+
+[Nick](https://www.dolthub.com/team#nick) has been hard at work making merges faster. The Dolt merge algorithm is tricky to get right. Nick does a good job explaining why in [his latest article](https://www.dolthub.com/blog/2025-07-16-announcing-fast-merge/). If you're not interested in [the details](https://www.dolthub.com/blog/2025-07-16-announcing-fast-merge/), enjoy the >1000X faster merges in some cases by grabbing [the latest Dolt release](https://github.com/dolthub/dolt/releases).
+
+### Doltgres Extensions
+
+Doltgres [now supports extensions](https://www.dolthub.com/blog/2025-07-14-loading-native-extensions-alpha/). We need to know which extensions you want supported so we can extend the API. Respond to this email if you want us to make a Postgres extension you use work in Doltgres.
+
+Until next week. As always, just reply to this email if you want to chat.
```

Comment:
It says youre out for a couple weeks, so this is wrong right? See Until two weeks from tomorrow?